hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/scheduler/SchedulerExceptionHandler.java

```diff
@@ -0,0 +1,18 @@
+package org.apache.mesos.hdfs.scheduler;
+
```

Comment:
it is necessary to have a class for this behavior... it is registered as an uncaught exception handler.  Would you like to see this as an inner class instead?